pallets/automation-time/rpc/src/lib.rs

```diff
@@ -112,6 +123,37 @@ where
 		})
 	}
 
+	fn get_time_automation_fees(
+		&self,
+		action: u8,
+		executions: u32,
+		at: Option<<Block as BlockT>::Hash>,
+	) -> RpcResult<NumberOrHex> {
+		let api = self.client.runtime_api();
+		let at = BlockId::hash(at.unwrap_or_else(||
+			// If the block hash is not supplied assume the best block.
+			self.client.info().best_hash));
+		let runtime_api_result =
+			api.get_time_automation_fees(&at, action, executions).map_err(|e| {
+				CallError::Custom(ErrorObject::owned(
+					Error::RuntimeError.into(),
+					"Unable to get time automation fees",
+					Some(e.to_string()),
+				))
+			})?;
+
+		let try_into_rpc_balance = |value: Balance| {
+			value.try_into().map_err(|_| {
```

Comment:
Was `NumberOrHex` a workaround for serialization with u128? What if you `try_into()` a `u64` instead? Since these are fees I expect they'll always be pretty small comparatively.

Reply:
This is u64 max: `1,844,674,407 TUR` we're definitely safe.

Reply:
ahaha, kk sounds good

Reply:
![image](https://user-images.githubusercontent.com/22332240/179834060-2dc5f6c3-24d4-4614-acdc-d4ba6884f1d4.png)

does indeed look better this way!